modules/dxp/apps/app-builder-workflow/app-builder-workflow-web/src/main/resources/META-INF/resources/js/pages/entry/EditEntry.es.js

```diff
@@ -92,6 +91,47 @@ export default function EditEntry({dataRecordId, redirect}) {
 			? Liferay.Language.get('edit-entry')
 			: Liferay.Language.get('add-entry');
 
+	const doFetch = useCallback(() => {
+		setLoading(true);
+
+		if (appWorkflowDefinitionId) {
+			if (isEdit) {
+				getItem(
+					`/o/portal-workflow-metrics/v1.0/processes/${appWorkflowDefinitionId}/instances`,
+					{classPKs: [dataRecordId]}
+				).then(({items}) => {
+					setLoading(false);
+
+					if (items.length > 0) {
```

Comment:
Hey @albinojunior

`if (items.length) {`
 
is enough?

Reply:
It's avoids undefined on items.pop() (and the following variables) and the code inside of the if isn't make sense when items is an empty array.

Reply:
It's just prevents a many validations of undefined variables.

Reply:
@inacionery makes sense?

Reply:
I mean `if (teste.length > 0) {` has the same result as `if (teste.length) {`?

Reply:
Yes, for "if" condition both has the same result.